### ISSUE TITLE
Component property is not recognize if it has a negative value

### ISSUE THREAD

**Issue by varoot** [ember-cli/eslint-plugin-ember#78]
```js
export default Component.extend({
  layout,
  count: 2,
  tabindex: -1,

  someComputedValue: computed.reads('count'),
});
```

this will generate an error saying that "someComputedValue" single-line function should be above the "tabindex" unknown property

**Comment by michalsnik** [comment]
Thanks for reporting this @varoot !

**Comment by michalsnik** [comment]
@varoot fixed in `v3.6.2` 🚀  Thanks @jbandura !
